nilearn/_utils/estimator_checks.py

```diff
@@ -1228,10 +1228,57 @@ def check_img_estimator_requires_y_none(estimator) -> None:
         estimator.fit(input_img, None)
     except ValueError as ve:
         if all(msg not in str(ve) for msg in expected_err_msgs):
-            raise ve
-
+            raise ve@ignore_warnings()
```

Comment:
Is the `@ignore_warnings()` decorator intended here or a typo ?

Reply:
Typo

Reply:
😸 

```suggestion
            raise ve
```